app/bundles/PageBundle/Tests/Controller/PageControllerFunctionalTest.php

```diff
@@ -103,4 +104,34 @@ public function testPageWithProject(): void
         $savedPage = $this->em->find(Page::class, $page->getId());
         $this->assertSame($project->getId(), $savedPage->getProjects()->first()->getId());
     }
+
+    public function testPageWithNullCustomHtmlIsUpdated(): void
+    {
+        $page = new Page();
+
+        $page->setTitle('Page A');
+        $page->setAlias('page-a');
+        $page->setTemplate('mautic_code_mode');
+
+        $this->em->persist($page);
+        $this->em->flush();
+
+        $pageId        = $page->getId();
+        $crawler       = $this->client->request(Request::METHOD_GET, '/s/pages/edit/'.$pageId);
+        $buttonCrawler = $crawler->selectButton('Save & Close');
+        $form          = $buttonCrawler->form();
+
+        $form['page[title]']->setValue('New Page');
+
+        $this->client->submit($form);
+
+        Assert::assertTrue($this->client->getResponse()->isOk());
```

Comment:
You get more useful debug output when using
```suggestion
        $this->assertResponseIsSuccessful();
```

Reply:
Fixed.